soda/cmd/generate/model_cmd.go

```diff
@@ -11,12 +11,14 @@ var modelCmdConfig struct {
 	SkipMigration bool
 	StructTag     string
 	MigrationType string
+	PackagePath   string
 }
 
 func init() {
 	ModelCmd.Flags().StringVarP(&modelCmdConfig.StructTag, "struct-tag", "", "json", "sets the struct tags for model (xml or json)")
 	ModelCmd.Flags().StringVarP(&modelCmdConfig.MigrationType, "migration-type", "", "fizz", "sets the type of migration files for model (sql or fizz)")
 	ModelCmd.Flags().BoolVarP(&modelCmdConfig.SkipMigration, "skip-migration", "s", false, "Skip creating a new fizz migration for this model.")
+	ModelCmd.Flags().StringVarP(&modelCmdConfig.PackagePath, "package-path", "a", "models", "set the path to the package the model will be created in")
```

Comment:
Also I'm not sure package-path is a good name for this feature, since we already have `path` (which is the migrations path, but should be refactored to `migrations-path`). How about `models-path`?

Reply:
Sure, I'm happy to change those.  I had picked "a" as the second letter (p**A**ckage), since "p" was already used.  I think the short forms are handy to have when running the command often, since there is quite a lot to type otherwise.  But it's fine either way.

And I agree that `models-path` is a better name.

I'll change both of those and update the PR.  Thanks for the feedback!